docs/section4/4-10.md

```diff
@@ -0,0 +1,39 @@
+
+**Rule ID:** 4-10  
+ 
+**Rule Description:** Schedules for HVAC fans that provide outdoor air for ventilation shall be cycled ON and OFF to meet heating and cooling loads during unoccupied hours excluding HVAC systems that meet Table G3.1-4 Schedules for the proposed building exceptions #s 2 and 3.  
+**Rule Assertion:** P-RMR = expected value                                           
+**Appendix G Section:** Table G3.1-4 Schedules for the proposed building excluding exceptions #s 2 and 3  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+**Applicability Checks:**  
+
+1. Not applicable to HVAC systems in which there are health and safety mandated minimum ventilation requirements during unoccupied hours or to HVAC systems primarily serving computer rooms.
+ 
+**Function Call:** 
+
+1. get_HVAC_systems_primarily_serving_comp_rooms
+2. get_zones_health_safety_vent_reqs()
+3. get_list_hvac_systems_associated_with_zone()  
+
+**Manual Check:** None 
+
+**Rule Logic**  
+**Applicability Check 1:** 
+- Get list of hvac systems that primarily serve computer rooms: `hvac_systems_primarily_serving_comp_rooms_list_p = get_HVAC_systems_primarily_serving_comp_rooms (P_RMR)`
+- Create list of inappliable hvac system starting with those primarily serving computer rooms: `inapplicable_zones_list_p = hvac_systems_primarily_serving_comp_rooms_list_p`
+- Get list of zones that are likely to have health and safety mandated minimum ventilation requirements during unoccupied hours: `zones_health_safety_unocc_list_p = get_zones_health_safety_vent_reqs(P_RMR)`
+- For each zone in the P_RMR: `for zone_p in P_RMR...Zone:`
+    - Check if zone is likely to have health and safety mandated minimum ventilation requirements during unoccupied hours: `if zone_p.id in zones_health_safety_unocc_list_p:`
+        - Get the hvac systems associated with the zone: `zone_hvav_sys_list_p = get_list_hvac_systems_associated_with_zone(P_RMR,zone_p.id)`
+        - Add hvac system list to the list of inapplicable hvac systems: `hvac_systems_primarily_serving_comp_rooms_list_p = hvac_systems_primarily_serving_comp_rooms_list_p.append(zone_hvav_sys_list_p)`             
+- For each hvac_p in P-RMR: `for each hvac_p in P_RMR...HeatingVentilationAirConditioningSystem:`                         
+    - Check if the hvac system is applicable to this check (not in the list of inapplicable systems): `if hvac_p not in inapplicable_zones_list_p:`
+        - Get hvac fan system unoccupied operation control: `operation_during_unoccupied_p = hvac_p.fan_systems[0].operation_during_unoccupied`  
+        **Rule Assertion:**
+        - Case 1: For each applicable hvac system, if fan system operation during unoccupied periods is cycling in the P_RMR: `if operation_during_unoccupied_p == “CYCLING”: PASS`
+        - Case 2: Else, for each applicable fan system: `Else: FAIL`
+
+**[Back](../_toc.md)**
```

Comment:
I think this RDS looks good, need to address the one comment on the inapplicable_zones_list_p tag and change it to inapplicable_hvac_systems_list_p